docs/running-on-kubernetes.md

```diff
@@ -1182,6 +1182,15 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
   <td>2.4.0</td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.driver.volumes.label.[VolumeType].[VolumeName].[LabelName]</code></td>
```

Comment:
Well this looks wrong to me because `label` could be interpreted as `[VolumeType]` in the existing pattern.

Can we follow `options` location like you did previously? I expected the following specifically
```
spark.kubernetes.driver.volumes.[VolumeType].[VolumeName].label.[LabelName]
```